test/setup/outputs.tf

```diff
@@ -68,6 +68,10 @@ output "monitoring_workspace_users" {
   value = var.group_email
 }
 
+output "project_prefix" {
```

Comment:
```suggestion
output "project_suffix" {
```

Reply:
this output need to be `project_prefix` because it is replacing this env var

https://github.com/terraform-google-modules/terraform-example-foundation/blob/d1a56d408cdb6ef95da51c685d6beaf5f6a0b07d/build/int.cloudbuild.yaml#L156

that is used to define which project prefix will be used in all steps